### PR TITLE
Feature/trainers

### PR DESCRIPTION
**Pull request by kgajowy**

### CODE REVIEW

File: src/shared/models/UserModel.js [hunk 1]
```diff
@@ -2,4 +2,12 @@ export class UserModel {
     email;
     name;
     nickname;
+
+    static fromBackend(data) {
```
Comment (Tuhaj) [inline src/shared/models/UserModel.js]: I think that a better name for this argument is `user` not the over generic `data`

File: src/topic/components/Topic.js [hunk 2]
```diff
@@ -3,16 +3,22 @@ import TopicDescription from './TopicDescription';
 import TrainersList from './TrainersList';
 import styled from 'styled-components';
 import Participants from './Participants';
+import PropTypes from 'prop-types';
```
Comment (Tuhaj) [inline src/topic/components/Topic.js]: eslint doesn't send a warning here? should be on top with styled-components (picky, but I'm just curious)

File: src/topic/components/TrainersList.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-            <li>
-                <Text>Piotr Zientara</Text>
-            </li>
-            <li>
-                <Text>Ula Kaźmierczyk</Text>
-            </li>
-            <li>
-                <Text>Kamil Gajowy</Text>
-            </li>
-        </ul>
```
Comment (Tuhaj) [inline src/topic/components/TrainersList.js]: such a dream team of trainers removed in favour of John Does 😭

File: src/topic/components/Topic.js (hunk 2 above)
Comment (kgajowy) [reply src/topic/components/Topic.js]: No a single warning. Could you please provide a source why it should be on top?

File: src/topic/components/TrainersList.js (hunk 3 above)
Comment (kgajowy) [reply src/topic/components/TrainersList.js]: Oh (John) Noes!

File: src/shared/models/UserModel.js (hunk 1 above)
Comment (kgajowy) [reply src/shared/models/UserModel.js]: Thanks for catching this! Fixed in [https://github.com/warsawjs/topics-manager/pull/23/commits/2b170b841919de548f8e2da932d2746aec19b448]

**Pull request merged**